core/templates/dev/head/pages/exploration_player/AnswerPopupDirective.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the answer popup.
+ */
+
+// This directive is unusual in that it should only be invoked indirectly, as
```

Comment:
Parts of this docstring seem to contain copy-paste errors. In what case is the state name argument relevant and why is it talking about feedback?

Reply:
Removed this file.